tella-locking-ui/src/main/res/values/colors.xml

```diff
@@ -17,4 +17,18 @@
     <color name="wa_calc_gray">#A4A2A2</color>
     <color name="wa_dark_green">#3C6E72</color>
     <color name="wa_calc_color">#545957</color>
+    <color name="calc_blue_skin_white">#EBFFFFFF</color>
+    <color name="calc_blue_skin_dark_blue">#333380FF</color>
+    <color name="calc_blue_skin_petrol_blue">#4285F4</color>
```

Comment:
We already have most of this color in the shared ui too, so make sure to clean this after the refactor :).